considerations.md

```diff
@@ -0,0 +1,136 @@
+# Extensibility
+
+Implementations that are reading/processing manifests MUST NOT generate an error if they encounter an unknown property.
+Instead they MUST ignore unknown properties.
+
+# Canonicalization
+
+* OCI Artifacts are [content-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage). See [descriptors](descriptor.md) for more.
+* One benefit of content-addressable storage is easy deduplication.
+* Many artifacts might depend on a particular blob, but there will only be one blob in the store.
+* With a different serialization, that same semantic blob would have a different hash, and if both versions of the blob are referenced there will be two blobs with the same semantic content.
+* To allow efficient storage, implementations serializing content for blobs SHOULD use a canonical serialization.
```

Comment:
Is this a requirement for the registry who's receiving the content or for the client who's generating the manifest? I don't think registries should be in the business of re-serializing content on behalf of customers. For example, if a manifest is copied between registries with the "wrong" serialization method, I would still expect it to have the same digest in the destination registry.

Reply:
The client. The registry can't modify content as the client may be eg adding a signature based on the hash that it created.

Reply:
the `considerations.md` file was [coped from the image-spec ](https://github.com/opencontainers/image-spec/blob/master/considerations.md) to keep the artifacts-spec decoupled from artifact specific specs, like the image-spec.
The content is unchanged.

Reply:
> the `considerations.md` file was [coped from the image-spec ](https://github.com/opencontainers/image-spec/blob/master/considerations.md) to keep the artifacts-spec decoupled from artifact specific specs, like the image-spec.
> The content is unchanged.

We thought about having copies in the distribution spec.. then thought better of it and just provided links.